Makefile

```diff
@@ -390,7 +390,8 @@ endif
 # On the PC, place quatlib in the directory ./quat.  No actual system
 # includes should be needed.
 ifeq ($(HW_OS),pc_cygwin)
-  INCLUDE_FLAGS := -I. -I./quat -I./atmellib -I./gpsnmealib
+  SYS_INCLUDE := -I./submodules/hidapi/hidapi -I/usr/include/libusb-1.0
```

Comment:
Do you mean for the `SYS_INCLUDE` entries to be [`-isystem`](https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html) instead of `-I`?

Reply:
This was written in the dim past, before -isystem.  It looks like the SYS_INCLUDE variable was being ignored in the Cygwin Makefile, so I hard-coded it in here to make it clear what was happening.

Thanks for checking on this.